Pod/Tests/TestStash.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import XCTest
```

Comment:
I think this should go to hyper-says:

testThisWeirdoThingyInTheContextWhenItShouldWorkAsDesired.